commit ac992937b461479b41f7f65e79646ef0a715f95d
Author: Brent Cook <busterb@gmail.com>
Date:   Thu Nov 9 12:29:33 2023 -0600

    Revert "aesni_ctr32_encrypt_blocks() is called indirectly from C code, so it"
    
    This reverts commit 52f96c842e82486510ffc2513e71323b70242f28.

diff --git a/src/lib/libcrypto/aes/asm/aesni-x86_64.pl b/src/lib/libcrypto/aes/asm/aesni-x86_64.pl
index 07d40a84a..c261a3795 100644
--- a/src/lib/libcrypto/aes/asm/aesni-x86_64.pl
+++ b/src/lib/libcrypto/aes/asm/aesni-x86_64.pl
@@ -1025,7 +1025,6 @@ $code.=<<___;
 .type	aesni_ctr32_encrypt_blocks,\@function,5
 .align	16
 aesni_ctr32_encrypt_blocks:
-	endbr64
 	lea	(%rsp),%rax
 	push	%rbp
 	sub	\$$frame_size,%rsp
